### ISSUE TITLE
New (automation) example for HASmartIrrigation v2.0 needed

### ISSUE THREAD

**Issue by Kurisutian** [rgc99/irrigation_unlimited#137]
With the recent update for HASmartIrrigation a lot of things have changed. So all examples and automation do not work any longer with the new version v2.0.
So the examples given to integrate HASmartIrrigation for calculating watering times have to be adjusted to fit with the new version to make it work again.
It would be great to have some example for the new version in the docs. I wanted to adjust it myself but somewhere I miss something in the logic and automation to fully understand it so I messed up. 

Thanks! 😄

**Comment by rgc99** [comment]
Yes it does look like HASmartirrigation has undergone a revamp. I haven't had the time to read through the new API's etc. @Kolia56 has done quite a bit of work integrating these two HA components. You can find his work [here](https://github.com/Kolia56/irrigation). Hope this helps.

Regards

**Comment by Kolia56** [comment]
I haven't updated to HASmartirrigation V2 yet, as I use only one zone, do not calculate ET nor precipitation (I have a PWS that delivers the data) and system is in production. As far as the code related to Irrigation Unlimited is concerned, the update would be rather straightforward. 
Replace
```smart_irrigation.[instance]_calculate_daily_adjusted_run_time```
by 
```Smart Irrigation: calculate_zone or Smart Irrigation: calculate_all_zones```
Replace 
```sensor.smart_irrigation_[instance]_daily_adjusted_run_time```
by 
```sensor.smart_irrigation_[zone_name]```
Replace 
```smart_irrigation.smart_irrigation_id7_reset_bucket```
by 
```Smart Irrigation: reset_bucket or Smart Irrigation: reset_all_buckets```
Then you should not be very far from a working solution. Depending on the number of zones you need to calculate irrigation time, you may need to do some home work.

**Comment by Detmud** [comment]
Hey, i am trying really hard but the hints are not helping. Does someone have a simple complete example i could start with `?
